File: c++/deleteAtIndex.cpp
// This is a C++ Language program to, Delete an element at a given index!

#include<iostream>
using namespace std;
int main(){
    int array[30] = {0};
    int n = 0, position = 0, index = 0;
    cout<<"Enter the Number of Elements you wants to insert:   ";
    cin>>n;
    cout<<"\n";
    
    cout<<"Enter each element one by one:  \n";
    for(int i=0; i<n; i++){
        cin>>array[i];
    }
    
    cout<<"Enter the position of element to be deleted:    ";
    cin>>position;
    cout<<"\n";
    
    index = position - 1;
    
    for(int i = index; i<n; i++){
        array[i] = array[i+1];
    }
    
    cout<<"Element Deleted!";
    for(int i=0; i<n-1; i++){
        cout<<array[i]<<"   ";
    }
    
    return 0;
}

File: python/deleteAtIndex.py
# Program to delete an element in the Array in Python Programming Language
array = [0]*30
index, position, value = 0, 0, 0

n = int(input("Enter the number of elements you wanted to insert:   "))
print("\n")

print("Enter each element one by one:   ")
print("\n")
for i in range(n):
    array[i] = int(input())

position = int(input("Enter position of element to be deleted:  "))
print("\n")

index = position - 1

for i in range(index,n+1):
    array[i] = array[i+1]

print("Element Deleted successfully!\n")
print("List of all the elements:    \n")
for i in range(n-1):
    print(array[i],end="\n")
    


